install.sh

```diff
@@ -11,32 +11,59 @@ waldo_exec2_name="sim_appcenter_build_and_upload.sh"
 waldo_asset1_name=
 waldo_asset2_name=
 
+waldo_exec_path=
+
 waldo_found_in_path=false
 waldo_is_reinstall=false
 
+function authorize() {
+    local _user_token=${WALDO_USER_TOKEN:-${TOKEN:-}}
+    local _auth_done=false
```

Comment:
Unused variable ?

Reply:
Good catch. A remnant of an earlier iteration.